src/rust/ide/view/src/project.rs

```diff
@@ -56,9 +57,26 @@ impl Model {
         Self{app,logger,display_object,graph_editor,searcher,code_editor}
     }
 
-    fn set_style(&self, is_light:bool) {
-        if is_light { self.app.themes.set_enabled(&["dark"])  }
-        else        { self.app.themes.set_enabled(&["light"]) }
+    /// Sets style of IDE to the one defined by parameter `theme`.
+    pub fn set_style(&self, theme: ensogl_theme::Theme) {
+        match theme {
+            ensogl_theme::Light => { self.set_light_style() },
+            ensogl_theme::Dark  => { self.set_dark_style()  },
+        }
+    }
+
+    fn set_light_style(&self) {
+        ensogl_theme::builtin::light::enable(&self.app);
+        self.set_html_style("light-theme");
+    }
+
+    fn set_dark_style(&self) {
+        ensogl_theme::builtin::dark::enable(&self.app);
+        self.set_html_style("dark-theme");
+    }
+
+    fn set_html_style(&self, style:&'static str) {
+        web::run_on_element_by_id(&self.logger,"root",|root| root.set_class_name(style));
```

Comment:
change the name to `with_element_by_id_or_warn` so it will match all names we use in our libs